data-structures-queues/take-2nd.js

```diff
@@ -0,0 +1,18 @@
+/* exported take2nd */
+
+function take2nd(queue) {
+  const newQ = queue;
```

Comment:
No need to create a `const` here, you can just use `queue` in the rest of the code